themes/src/main/resources/theme/keycloak-preview/account/resources/app/content/my-resources-page/ShareTheResource.tsx

```diff
@@ -99,7 +99,7 @@ export class ShareTheResource extends React.Component<ShareTheResourceProps, Sha
 
         AccountServiceClient.Instance.doPut('/resources/' + rscId + '/permissions', {data: permissions})
             .then(() => {
-                ContentAlert.success(Msg.localize('shareSuccess'));
+                ContentAlert.success('shareSuccess');
```

Comment:
We still need Msg.localize here right?
```suggestion
                ContentAlert.success(Msg.localize('shareSuccess'));
```

Reply:
No.  ContentAlert calls Msg.localize internally.  So you can just pass a key or a literal and it will take care of it.

Reply:
Right, that makes sense